### ISSUE TITLE
2025-05-26, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#181]
# Today's Arxiv Papers

(1) [The Staircase of Ethics: Probing LLM Value Priorities through Multi-Step Induction to Complex Moral Dilemmas](http://arxiv.org/abs/2505.18154v1), 2025-05-23

(2) [Fann or Flop: A Multigenre, Multiera Benchmark for Arabic Poetry Understanding in LLMs](http://arxiv.org/abs/2505.18152v1), 2025-05-23

(3) [First Finish Search: Efficient Test-Time Scaling in Large Language Models](http://arxiv.org/abs/2505.18149v1), 2025-05-23

(4) [Lost in the Haystack: Smaller Needles are More Difficult for LLMs to Find](http://arxiv.org/abs/2505.18148v1), 2025-05-23

(5) [Gaming Tool Preferences in Agentic LLMs](http://arxiv.org/abs/2505.18135v1), 2025-05-23

(6) [Frankentext: Stitching random text fragments into long-form narratives](http://arxiv.org/abs/2505.18128v1), 2025-05-23

(7) [Reward Model Overoptimisation in Iterated RLHF](http://arxiv.org/abs/2505.18126v1), 2025-05-23

(8) [UNJOIN: Enhancing Multi-Table Text-to-SQL Generation via Schema Simplification](http://arxiv.org/abs/2505.18122v1), 2025-05-23

(9) [ProgRM: Build Better GUI Agents with Progress Rewards](http://arxiv.org/abs/2505.18121v1), 2025-05-23

(10) [Bidirectional Knowledge Distillation for Enhancing Sequential Recommendation with Large Language Models](http://arxiv.org/abs/2505.18120v1), 2025-05-23

(11) [Bridging Supervised Learning and Reinforcement Learning in Math Reasoning](http://arxiv.org/abs/2505.18116v1), 2025-05-23

(12) [Instructify: Demystifying Metadata to Visual Instruction Tuning Data Conversion](http://arxiv.org/abs/2505.18115v1), 2025-05-23

(13) [Watch and Listen: Understanding Audio-Visual-Speech Moments with Multimodal LLM](http://arxiv.org/abs/2505.18110v1), 2025-05-23

(14) [ManuSearch: Democratizing Deep Search in Large Language Models with a Transparent and Open Multi-Agent Framework](http://arxiv.org/abs/2505.18105v1), 2025-05-23

(15) [How Can I Publish My LLM Benchmark Without Giving the True Answers Away?](http://arxiv.org/abs/2505.18102v1), 2025-05-23

(16) [Planning without Search: Refining Frontier LLMs with Offline Goal-Conditioned RL](http://arxiv.org/abs/2505.18098v1), 2025-05-23

(17) [QwenLong-CPRS: Towards $\infty$-LLMs with Dynamic Context Optimization](http://arxiv.org/abs/2505.18092v1), 2025-05-23

(18) [Data Mixing Can Induce Phase Transitions in Knowledge Acquisition](http://arxiv.org/abs/2505.18091v1), 2025-05-23

(19) [Stable Reinforcement Learning for Efficient Reasoning](http://arxiv.org/abs/2505.18086v1), 2025-05-23

(20) [Deep Video Discovery: Agentic Search with Tool Use for Long-form Video Understanding](http://arxiv.org/abs/2505.18079v1), 2025-05-23

(21) [Extended Inductive Reasoning for Personalized Preference Inference from Behavioral Signals](http://arxiv.org/abs/2505.18071v1), 2025-05-23

(22) [Emergence of Hebbian Dynamics in Regularized Non-Local Learners](http://arxiv.org/abs/2505.18069v1), 2025-05-23

(23) [Reward Model Generalization for Compute-Aware Test-Time Reasoning](http://arxiv.org/abs/2505.18065v1), 2025-05-23

(24) [MathEDU: Towards Adaptive Feedback for Student Mathematical Problem-Solving](http://arxiv.org/abs/2505.18056v1), 2025-05-23

(25) [Contrastive Distillation of Emotion Knowledge from LLMs for Zero-Shot Emotion Recognition](http://arxiv.org/abs/2505.18040v1), 2025-05-23

(26) [Structured Thinking Matters: Improving LLMs Generalization in Causal Inference Tasks](http://arxiv.org/abs/2505.18034v1), 2025-05-23

(27) [LLM assisted web application functional requirements generation: A case study of four popular LLMs over a Mess Management System](http://arxiv.org/abs/2505.18019v1), 2025-05-23

(28) [Training with Pseudo-Code for Instruction Following](http://arxiv.org/abs/2505.18011v1), 2025-05-23

(29) [Towards Analyzing and Understanding the Limitations of VAPO: A Theoretical Perspective](http://arxiv.org/abs/2505.17997v1), 2025-05-23

(30) [Outcome-based Reinforcement Learning to Predict the Future](http://arxiv.org/abs/2505.17989v1), 2025-05-23

(31) [Towards Revealing the Effectiveness of Small-Scale Fine-tuning in R1-style Reinforcement Learning](http://arxiv.org/abs/2505.17988v1), 2025-05-23

(32) [ADLGen: Synthesizing Symbolic, Event-Triggered Sensor Sequences for Human Activity Modeling](http://arxiv.org/abs/2505.17987v1), 2025-05-23

(33) [SmartNote: An LLM-Powered, Personalised Release Note Generator That Just Works](http://arxiv.org/abs/2505.17977v1), 2025-05-23

(34) [Generalized Fisher-Weighted SVD: Scalable Kronecker-Factored Fisher Approximation for Compressing Large Language Models](http://arxiv.org/abs/2505.17974v1), 2025-05-23

(35) [Are Large Language Models Reliable AI Scientists? Assessing Reverse-Engineering of Black-Box Systems](http://arxiv.org/abs/2505.17968v1), 2025-05-23

(36) [SVD-Free Low-Rank Adaptive Gradient Optimization for Large Language Models](http://arxiv.org/abs/2505.17967v1), 2025-05-23

(37) [Beyond Distillation: Pushing the Limits of Medical LLM Reasoning with Minimalist Rule-Based RL](http://arxiv.org/abs/2505.17952v1), 2025-05-23

(38) [Survival Games: Human-LLM Strategic Showdowns under Severe Resource Scarcity](http://arxiv.org/abs/2505.17937v1), 2025-05-23

(39) [Towards Practical Defect-Focused Automated Code Review](http://arxiv.org/abs/2505.17928v1), 2025-05-23

(40) [LLM Meeting Decision Trees on Tabular Data](http://arxiv.org/abs/2505.17918v1), 2025-05-23

(41) [T2I-Eval-R1: Reinforcement Learning-Driven Reasoning for Interpretable Text-to-Image Evaluation](http://arxiv.org/abs/2505.17897v1), 2025-05-23

(42) [Mutarjim: Advancing Bidirectional Arabic-English Translation with a Small Language Model](http://arxiv.org/abs/2505.17894v1), 2025-05-23

(43) [LLM4SP: Large Language Models for Scatterer Prediction via Synesthesia of Machines](http://arxiv.org/abs/2505.17879v1), 2025-05-23

(44) [MOOSE-Chem3: Toward Experiment-Guided Hypothesis Ranking via Simulated Experimental Feedback](http://arxiv.org/abs/2505.17873v1), 2025-05-23

(45) [The emergence of sparse attention: impact of data distribution and benefits of repetition](http://arxiv.org/abs/2505.17863v1), 2025-05-23

(46) [Daily-Omni: Towards Audio-Visual Reasoning with Temporal Alignment across Modalities](http://arxiv.org/abs/2505.17862v1), 2025-05-23

(47) [Superplatforms Have to Attack AI Agents](http://arxiv.org/abs/2505.17861v1), 2025-05-23

(48) [Automated Testing of the GUI of a Real-Life Engineering Software using Large Language Models](http://dx.doi.org/10.1109/ICSTW64639.2025.10962502), 2025-05-23

(49) [Stepwise Reasoning Checkpoint Analysis: A Test Time Scaling Method to Enhance LLMs' Reasoning](http://arxiv.org/abs/2505.17829v1), 2025-05-23

(50) [Trinity-RFT: A General-Purpose and Unified Framework for Reinforcement Fine-Tuning of Large Language Models](http://arxiv.org/abs/2505.17826v1), 2025-05-23

(51) [PatientSim: A Persona-Driven Simulator for Realistic Doctor-Patient Interactions](http://arxiv.org/abs/2505.17818v1), 2025-05-23

(52) [Don't Overthink it. Preferring Shorter Thinking Chains for Improved LLM Reasoning](http://arxiv.org/abs/2505.17813v1), 2025-05-23

(53) [Integrating Counterfactual Simulations with Language Models for Explaining Multi-Agent Behaviour](http://arxiv.org/abs/2505.17801v1), 2025-05-23

(54) [DialogXpert: Driving Intelligent and Emotion-Aware Conversations through Online Value-Based Reinforcement Learning with LLM Priors](http://arxiv.org/abs/2505.17795v1), 2025-05-23

(55) [RECIPE-TKG: From Sparse History to Structured Reasoning for LLM-based Temporal Knowledge Graph Completion](http://arxiv.org/abs/2505.17794v1), 2025-05-23

(56) [Titanus: Enabling KV Cache Pruning and Quantization On-the-Fly for LLM Acceleration](http://arxiv.org/abs/2505.17787v1), 2025-05-23

(57) [EXECUTE: A Multilingual Benchmark for LLM Token Understanding](http://arxiv.org/abs/2505.17784v1), 2025-05-23

(58) [C-LoRA: Contextual Low-Rank Adaptation for Uncertainty Estimation in Large Language Models](http://arxiv.org/abs/2505.17773v1), 2025-05-23

(59) [Inference-Time Decomposition of Activations (ITDA): A Scalable Approach to Interpreting Large Language Models](http://arxiv.org/abs/2505.17769v1), 2025-05-23

(60) [R-Genie: Reasoning-Guided Generative Image Editing](http://arxiv.org/abs/2505.17768v1), 2025-05-23

(61) [The Real Barrier to LLM Agent Usability is Agentic ROI](http://arxiv.org/abs/2505.17767v1), 2025-05-23

(62) [Resolving Conflicting Evidence in Automated Fact-Checking: A Study on Retrieval-Augmented LLMs](http://arxiv.org/abs/2505.17762v1), 2025-05-23

(63) [But what is your honest answer? Aiding LLM-judges with honest alternatives using steering vectors](http://arxiv.org/abs/2505.17760v1), 2025-05-23

(64) [Fast Quiet-STaR: Thinking Without Thought Tokens](http://arxiv.org/abs/2505.17746v1), 2025-05-23

(65) [Automating Safety Enhancement for LLM-based Agents with Synthetic Risk Scenarios](http://arxiv.org/abs/2505.17735v1), 2025-05-23

(66) [Slot-MLLM: Object-Centric Visual Tokenization for Multimodal LLM](http://arxiv.org/abs/2505.17726v1), 2025-05-23

(67) [Get Experience from Practice: LLM Agents with Record & Replay](http://arxiv.org/abs/2505.17716v1), 2025-05-23

(68) [Understanding How Value Neurons Shape the Generation of Specified Values in LLMs](http://arxiv.org/abs/2505.17712v1), 2025-05-23

(69) [LLM Contribution Summarization in Software Projects](http://arxiv.org/abs/2505.17710v1), 2025-05-23

(70) [CIKT: A Collaborative and Iterative Knowledge Tracing Framework with Large Language Models](http://arxiv.org/abs/2505.17705v1), 2025-05-23

(71) [Seek-CAD: A Self-refined Generative Modeling for 3D Parametric CAD Using Local Inference via DeepSeek](http://arxiv.org/abs/2505.17702v1), 2025-05-23

(72) [COUNTDOWN: Contextually Sparse Activation Filtering Out Unnecessary Weights in Down Projection](http://arxiv.org/abs/2505.17701v1), 2025-05-23

(73) [Activation Control for Efficiently Eliciting Long Chain-of-thought Ability of Language Models](http://arxiv.org/abs/2505.17697v1), 2025-05-23

(74) [FlashForge: Ultra-Efficient Prefix-Aware Attention for LLM Decoding](http://arxiv.org/abs/2505.17694v1), 2025-05-23

(75) [ELSPR: Evaluator LLM Training Data Self-Purification on Non-Transitive Preferences via Tournament Graph Reconstruction](http://arxiv.org/abs/2505.17691v1), 2025-05-23

(76) [Tuning Language Models for Robust Prediction of Diverse User Behaviors](http://arxiv.org/abs/2505.17682v1), 2025-05-23

(77) [Rethinking Agent Design: From Top-Down Workflows to Bottom-Up Skill Evolution](http://arxiv.org/abs/2505.17673v1), 2025-05-23

(78) [MIDB: Multilingual Instruction Data Booster for Enhancing Multilingual Instruction Synthesis](http://arxiv.org/abs/2505.17671v1), 2025-05-23

(79) [Towards Dynamic Theory of Mind: Evaluating LLM Adaptation to Temporal Evolution of Human States](http://arxiv.org/abs/2505.17663v1), 2025-05-23

(80) [Plan-R1: Safe and Feasible Trajectory Planning as Language Modeling](http://arxiv.org/abs/2505.17659v1), 2025-05-23

(81) [Too Consistent to Detect: A Study of Self-Consistent Errors in LLMs](http://arxiv.org/abs/2505.17656v1), 2025-05-23

(82) [EVADE: Multimodal Benchmark for Evasive Content Detection in E-Commerce Applications](http://arxiv.org/abs/2505.17654v1), 2025-05-23

(83) [GeoGramBench: Benchmarking the Geometric Program Reasoning in Modern LLMs](http://arxiv.org/abs/2505.17653v1), 2025-05-23

(84) [Rethinking the Sampling Criteria in Reinforcement Learning for LLM Reasoning: A Competence-Difficulty Alignment Perspective](http://arxiv.org/abs/2505.17652v1), 2025-05-23

(85) [Simulating Macroeconomic Expectations using LLM Agents](http://arxiv.org/abs/2505.17648v1), 2025-05-23

(86) [Understanding Pre-training and Fine-tuning from Loss Landscape Perspectives](http://arxiv.org/abs/2505.17646v1), 2025-05-23

(87) [HoloLLM: Multisensory Foundation Model for Language-Grounded Human Sensing and Reasoning](http://arxiv.org/abs/2505.17645v1), 2025-05-23

(88) [PreMoe: Lightening MoEs on Constrained Memory by Expert Pruning and Retrieval](http://arxiv.org/abs/2505.17639v1), 2025-05-23

(89) [Surfacing Semantic Orthogonality Across Model Safety Benchmarks: A Multi-Dimensional Analysis](http://dx.doi.org/10.5121/csit.2025.150903), 2025-05-23

(90) [ReqBrain: Task-Specific Instruction Tuning of LLMs for AI-Assisted Requirements Generation](http://arxiv.org/abs/2505.17632v1), 2025-05-23

(91) [GIM: Improved Interpretability for Large Language Models](http://arxiv.org/abs/2505.17630v1), 2025-05-23

(92) [Enhancing Large Vision-Language Models with Layout Modality for Table Question Answering on Japanese Annual Securities Reports](http://arxiv.org/abs/2505.17625v1), 2025-05-23

(93) [Navigate the Unknown: Enhancing LLM Reasoning with Intrinsic Motivation Guided Exploration](http://arxiv.org/abs/2505.17621v1), 2025-05-23

(94) [Runaway is Ashamed, But Helpful: On the Early-Exit Behavior of Large Language Model-based Agents in Embodied Environments](http://arxiv.org/abs/2505.17616v1), 2025-05-23

(95) [Large language model as user daily behavior data generator: balancing population diversity and individual personality](http://arxiv.org/abs/2505.17615v1), 2025-05-23

(96) [Distilling LLM Agent into Small Models with Retrieval and Code Tools](http://arxiv.org/abs/2505.17612v1), 2025-05-23

(97) [Decoupled Visual Interpretation and Linguistic Reasoning for Math Problem Solving](http://arxiv.org/abs/2505.17609v1), 2025-05-23

(98) [Controlled Agentic Planning & Reasoning for Mechanism Synthesis](http://arxiv.org/abs/2505.17607v1), 2025-05-23

(99) [Wolf Hidden in Sheep's Conversations: Toward Harmless Data-Based Backdoor Attacks for Jailbreaking Large Language Models](http://arxiv.org/abs/2505.17601v1), 2025-05-23

(100) [Dynamic Text Bundling Supervision for Zero-Shot Inference on Text-Attributed Graphs](http://arxiv.org/abs/2505.17599v1), 2025-05-23
